### DISCUSSION TITLE
we need a Z icon for zen browser ...

### DISCUSSION THREAD

**Discussion by Any-Who** [zen-browser/desktop#3587] [General]
please put the Z icon in package somewhere so i can use it for zen icon. i don't like the circle cuz it makes me feel like i'm shopping at target ....

**Comment by astudentinearth** [comment]
I think the Z was Zen's identity, it was unique in its looks. I hope it comes back

**Reply by universish** [reply]
https://github.com/zen-browser/desktop/issues/2870#issuecomment-2537891646

**Reply by KakouMoses** [reply]
I hope it too...
